src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -129,36 +141,40 @@
            :else [:div {:style {:textAlign "center"}}
                   [comps/Spinner {:text "Loading Method Configuration..."}]]))
    :component-did-mount
-   (fn [{:keys [props state this]}]
+   (fn [{:keys [state this props]}]
      (this :-load-validated-method-config)
      (endpoints/call-ajax-orch
-      {:endpoint endpoints/list-methods
+      {:endpoint (endpoints/list-method-snapshots (get-in props [:config-id :namespace]))
```

Comment:
the call to `-load-validated-method-config` just above this returns the 
```
"methodRepoMethod": {
      "methodNamespace": "davidan2",
      "methodName": "foredit",
      "methodVersion": -1
    }
```
block, which has the namespace/name you'd need to pass to `list-method-snapshots` for a truly targeted call. Of course this means the two ajax calls would be serialized.